src/manual/task-execution/provisioning.md

```diff
@@ -0,0 +1,46 @@
+---
+filename: task-execution/provisioning.md
+title: Provisioning
+order: 40
+---
+
+In the simplest case, a worker implementation is installed and configured
+manually on a host and executes tasks indefinitely. For most purposes, it is
+far more cost-effective to dynamically adjust the pool of available workers
+based on the work to be done. This process is called provisioning.
+
+A provisioner operates by monitoring the pool of running workers and the
+corresponding queues. Based on the queue length and the configuration of the
+provisioner, it may create new workers or signal to running workers that they
+should stop. Of course, provisioners might take into account additional
```

Comment:
The AWS provisioner doesn't signal to running workers that they should stop - it only has a 96 hour spot-kill safeguard.